EventFilter/CTPPSRawToDigi/interface/RPixErrorChecker.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RPixErrorChecker_H
+#define RPixErrorChecker_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h